### DISCUSSION TITLE
One year of Cruiser!

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#358] [Show and tell]
I've been a Cruisianer for exactly one year - quasi day one customer after the discontinue of the Kurviger app.
At first, I just wanted to make sure that I could continue to navigate my Kurviger plans. But the many improvements and new features have convinced me: although my Kurviger app subscription does not expire until the end of 2023, I have been using Cruiser exclusively for a year now! 
**Thank you emux - a truly outstanding job!** 

<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/e5186879-359d-4d6e-b7ba-97602919c3d4"/>

**Comment by devemux86** [comment]
@WolfgangZ42 many thanks for the kind words and the support!

Please keep pushing Cruiser to its limits and report anything you find.

The journey goes on with many new features in the future. :slightly_smiling_face:

**Comment by Cer-o** [comment]
También provengo de Kurviger y su nueva cara no me termina de convencer. 
Probé Cruiser buscando algo parecido al viejo Kurviger y me ha convencido tanto que la he adoptado como mi compañera de viaje en exclusiva.
Soy un usuario básico y doy máxima importancia a la sencillez de uso.

Desde aquí también quiero felicitar a @devemux86 por su extraordinario y CERCANO trabajo. Es muy de agradecer.
Un abrazo.

**Reply by devemux86** [reply]
@Cer-o 

Thank you for your kind words and support.
If you find anything complicated, please let me know and I'm sure we can make it better!

Gracias por tus amables palabras y por tu apoyo.
Si encuentras algo complicado, por favor, házmelo saber y ¡seguro que podemos mejorarlo!
